cryostatio/cryostat-web
# chore(eslint): add eslint-webpack-plugin

Signed-off-by: Max Cao <macao@redhat.com>

# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
_______________________________________________

Fixes: #841 

## Description of the change:
This change allows eslint errors and warnings to be shown during the webpack development compilation build status.
This only works during a development build using `yarn start:dev`

## Motivation for the change:
See #841 

## How to manually test:
1. checkout pr and `yarn install`
2. Run cryostat with this pr -web client with `yarn start:dev`
3. Go to a file like AboutAgentCard.tsx in `src/app/Agent/AboutAgentCard.tsx` 
4. Change the import order of the 2 imports
5. Notice the webpack eslint warning
